src/docstring_template.jl

```diff
@@ -61,12 +61,38 @@ function _docstring(cfg_symb::Symbol)
         $(out)
         """
 end
-#= 
-""" 
-    $(TYPEDSIGNATURES)
-$(_docstring(:erp))
 
-"""
-function plot_new()
-    return "b"
-end =#
+function indiv_docstrings(cfg_symb::Symbol)
```

Comment:
do you use the same function to actually define the defaults? else you have them double written anyway; in that case I'd prefer to have them directly in the docstring, close to the function defining it

Reply:
no, they are set only once